tests/calendar-builder.test.ts

```diff
@@ -204,6 +208,96 @@ describe('Testing calendar generation functions', () => {
     });
   });
 
+  describe('Testing calendar metadata', () => {
+    const testCalendarMetada = async (scope: CalendarScope) => {
```

Comment:
Is it intentional to name this variable `testCalendarMetada`? Didn’t you want to name it `testCalendarMetadata`? :wink:

Reply:
Yes, I forgot a syllable. Thanks!